docs/index.html

```diff
@@ -2,191 +2,615 @@
 layout: default
 title: eBay Skin
 ds: 6
+# Schema Proposal 1
```

Comment:
Schema 1 - module has some DS fields, optional status field, and submodules field.